test/devices_tests/binary_sensor_test.py

```diff
@@ -11,6 +10,11 @@
 from xknx.telegram import GroupAddress, Telegram
 
 
+class AsyncMock(MagicMock):
```

Comment:
AsyncMock was added to `unittest.mock` only after 2.8 and was then made the default. When we drop 3.6.and 3.7 support we can remove this again.